packages/connect-node/src/connect-protocol.ts

```diff
@@ -92,6 +96,18 @@ export function createConnectProtocol(
                 "Unsupported Media Type"
               );
             }
+
+            if (
+              shouldReject &&
+              (req.headers[connectProtocolVersionHeader] ?? "").length === 0
+            ) {
```

Comment:
As @pkwarren and @jchadwick-buf [pointed out](https://github.com/bufbuild/connect-go/pull/416#discussion_r1043537408), we should require the only specified value, `"1"`.